docs/guides/io_config.mdx

```diff
@@ -116,4 +116,5 @@ default:
     source: trino-cli                         # Optional
     user: admin
     verify: /path/to/your/ca.crt              # Optional
+    precision: 3                              # Optional. Set TIMESTAMP(n) precision
```

Comment:
let's put it under `data_type_propereties` like:
```
data_type_propereties
  timestamp_precision
``

Reply:
Modified on both .mdx and code in 3e4566b7fb71b918ec79c0fff3164eaed6e8f545